cmd/krel/cmd/sign_blobs.go

```diff
@@ -123,16 +123,17 @@ func runSignBlobs(signOpts *signOptions, signBlobOpts *signBlobOptions, args []s
 				strings.Contains(file, "README") || strings.Contains(file, "Makefile") {
 				continue
 			}
-			temp := strings.TrimPrefix(file, object.GcsPrefix)
-			err = gcsClient.CopyToLocal(file, tempDir)
+			destinationPath := strings.TrimPrefix(file, object.GcsPrefix)
+			localPath := fmt.Sprintf("%s/%s/%s", tempDir, filepath.Dir(destinationPath), filepath.Base(destinationPath))
```

Comment:
Is it better to use `filepath.Join` here?

Reply:
true, forgot that
thanks!